tests/lora/test_lora_layers_old_backend.py

```diff
@@ -1550,11 +1550,12 @@ def test_lora_on_off(self, expected_max_diff=1e-3):
         assert max_diff_new_sample < expected_max_diff
         assert max_diff_old_sample < expected_max_diff
 
+    @unittest.skip("Difference is too large. Test is not useful")
```

Comment:
Hmmm that sounds like a bug then. Actually this is highly likely related to https://github.com/huggingface/diffusers/issues/5135

Reply:
```suggestion
```

Let's remove the `@unittest.skip(...)` part, but feel free to merge the PR otherwise and fix xfromers lora in a follow-up